doc/api/worker_threads.md

```diff
@@ -517,8 +520,8 @@ changes:
     description: The `argv` option was introduced.
 -->
 
-* `filename` {string} The path to the Worker’s main script. Must be
-  either an absolute path or a relative path (i.e. relative to the
+* `filename` {string|URL} The path to the Worker’s main script or module. Must
+  be either an absolute path or a relative path (i.e. relative to the
```

Comment:
This should be documented more clearly – the `|URL` addition makes it seem like this adds support for `URL` *objects*, but more importantly it adds support for `file://` URL *strings*. Imo it should be called out very explicitly that strings starting with `file://` will be handled differently from other strings, especially as the former are also valid file system paths.

(When accepting untrusted filenames from outside sources, this can present a security risk, as `file://` URLs look like relative paths but act like absolute ones – that’s the reason the `fs` module doesn’t support `file://` URL strings.)